In [20]:
# All Libraries required

!pip install numpy==2.0.2
!pip install pandas==2.2.2
!pip install tensorflow_cpu==2.18.0
!pip install scikit-learn==1.3.1

  Using cached numpy-2.0.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
Using cached numpy-2.0.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (19.5 MB)
  Attempting uninstall: numpy
    Found existing installation: numpy 1.26.4
    Uninstalling numpy-1.26.4:
      Successfully uninstalled numpy-1.26.4
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
cupy-cuda12x 12.2.0 requires numpy<1.27,>=1.20, but you have numpy 2.0.2 which is incompatible.
gensim 4.3.3 requires numpy<2.0,>=1.18.5, but you have numpy 2.0.2 which is incompatible.
langchain 0.3.12 requires numpy<2,>=1.22.4; python_version < "3.12", but you have numpy 2.0.2 which is incompatible.
matplotlib 3.8.0 requires numpy<2,>=1.21, but you have numpy 2.0.2 which is incompatible.
pytensor 2.26.4 requires numpy<2,>=1.17.0, but you have numpy 2.0.2 which is incompa

In [21]:
import pandas as pd
import numpy as np
from sklearn.model_selection import train_test_split
from sklearn.metrics import mean_squared_error
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense

In [30]:
# Load dataset
data = pd.read_csv("/content/drive/MyDrive/ML/concrete_data.csv")

In [31]:
# Split features and target
X = data.drop(columns=["Strength"])
y = data["Strength"]

In [32]:
# Normalize the predictors manually using the formula
X_normalized = (X - X.mean()) / X.std()

In [25]:
print(X_normalized)

        Cement  Blast Furnace Slag   Fly Ash     Water  Superplasticizer  \
0     2.476712           -0.856472 -0.846733 -0.916319         -0.620147   
1     2.476712           -0.856472 -0.846733 -0.916319         -0.620147   
2     0.491187            0.795140 -0.846733  2.174405         -1.038638   
3     0.491187            0.795140 -0.846733  2.174405         -1.038638   
4    -0.790075            0.678079 -0.846733  0.488555         -1.038638   
...        ...                 ...       ...       ...               ...   
1025 -0.045623            0.487998  0.564271 -0.092126          0.451190   
1026  0.392628           -0.856472  0.959602  0.675872          0.702285   
1027 -1.269472            0.759210  0.850222  0.521336         -0.017520   
1028 -1.168042            1.307430 -0.846733 -0.279443          0.852942   
1029 -0.193939            0.308349  0.376762  0.891286          0.400971   

      Coarse Aggregate  Fine Aggregate       Age  
0             0.862735       -1.2170

In [33]:
# Initialize list to store MSE values
mse_list = []

# Repeat 50 times
for _ in range(50):
    # 1. Split the data into training and test sets
    X_train, X_test, y_train, y_test = train_test_split(X_normalized, y, test_size=0.3, random_state=np.random.randint(0, 1000))

    # 2. Build the model
    model = Sequential()
    model.add(Dense(10, activation='relu', input_shape=(X_train.shape[1],)))
    model.add(Dense(1))

    # Compile the model
    model.compile(optimizer='adam', loss='mean_squared_error')

    # Train the model
    model.fit(X_train, y_train, validation_data=(X_test, y_test), epochs=100, batch_size=10, verbose=2)

    # 3. Evaluate the model
    y_pred = model.predict(X_test)
    mse = mean_squared_error(y_test, y_pred)

    # Store MSE
    mse_list.append(mse)

Epoch 1/100


/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1634.6860 - val_loss: 1479.5114
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1594.3662 - val_loss: 1445.5970
Epoch 3/100
73/73 - 0s - 3ms/step - loss: 1558.6063 - val_loss: 1413.5592
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1521.5579 - val_loss: 1377.9753
Epoch 5/100
73/73 - 0s - 3ms/step - loss: 1480.6033 - val_loss: 1337.4692
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1432.9224 - val_loss: 1289.8729
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1377.4747 - val_loss: 1236.4517
Epoch 8/100
73/73 - 0s - 2ms/step - loss: 1315.8844 - val_loss: 1177.7065
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1247.9115 - val_loss: 1113.5631
Epoch 10/100
73/73 - 0s - 5ms/step - loss: 1176.2360 - val_loss: 1046.7496
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 1097.3192 - val_loss: 973.2015
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 1009.9346 - val_loss: 892.9365
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 917.4434 - val_loss: 809.5549
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 821.03

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1569.0685 - val_loss: 1517.6445
Epoch 2/100
73/73 - 0s - 2ms/step - loss: 1522.5874 - val_loss: 1467.5442
Epoch 3/100
73/73 - 0s - 5ms/step - loss: 1473.9209 - val_loss: 1413.8564
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1419.7950 - val_loss: 1355.1818
Epoch 5/100
73/73 - 0s - 3ms/step - loss: 1360.9001 - val_loss: 1290.2123
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1295.8492 - val_loss: 1222.3352
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1226.9541 - val_loss: 1148.9733
Epoch 8/100
73/73 - 0s - 5ms/step - loss: 1152.0651 - val_loss: 1072.5271
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1075.1205 - val_loss: 992.9958
Epoch 10/100
73/73 - 0s - 2ms/step - loss: 995.2889 - val_loss: 910.9996
Epoch 11/100
73/73 - 0s - 3ms/step - loss: 910.9402 - val_loss: 826.8669
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 828.1947 - val_loss: 745.0216
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 745.9061 - val_loss: 663.9512
Epoch 14/100
73/73 - 0s - 5ms/step - loss: 662.5747 - 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1548.5083 - val_loss: 1525.7179
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1496.3309 - val_loss: 1475.4673
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1441.6365 - val_loss: 1421.8655
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1383.4429 - val_loss: 1363.2072
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1318.7717 - val_loss: 1299.2897
Epoch 6/100
73/73 - 0s - 2ms/step - loss: 1248.7474 - val_loss: 1228.5933
Epoch 7/100
73/73 - 0s - 5ms/step - loss: 1173.4626 - val_loss: 1154.5315
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1094.0435 - val_loss: 1077.9786
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1011.7054 - val_loss: 998.6870
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 930.4458 - val_loss: 919.9581
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 849.7143 - val_loss: 843.5028
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 771.4341 - val_loss: 769.1458
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 694.8444 - val_loss: 696.9513
Epoch 14/100
73/73 - 0s - 5ms/step - loss: 623.3903 - 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1557.4509 - val_loss: 1561.0601
Epoch 2/100
73/73 - 0s - 2ms/step - loss: 1511.2042 - val_loss: 1507.6099
Epoch 3/100
73/73 - 0s - 5ms/step - loss: 1458.6222 - val_loss: 1447.7834
Epoch 4/100
73/73 - 0s - 3ms/step - loss: 1399.0659 - val_loss: 1379.6798
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1331.2664 - val_loss: 1304.3279
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1256.4620 - val_loss: 1221.7266
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1172.8812 - val_loss: 1131.0020
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1080.9827 - val_loss: 1031.3104
Epoch 9/100
73/73 - 0s - 3ms/step - loss: 981.0052 - val_loss: 928.5243
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 882.5772 - val_loss: 827.7103
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 784.2097 - val_loss: 728.6624
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 692.6359 - val_loss: 639.8469
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 606.0095 - val_loss: 556.3471
Epoch 14/100
73/73 - 0s - 5ms/step - loss: 526.6737 - v

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 16ms/step - loss: 1587.6283 - val_loss: 1617.5552
Epoch 2/100
73/73 - 0s - 5ms/step - loss: 1549.1354 - val_loss: 1579.6820
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1511.7777 - val_loss: 1540.7692
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1469.8136 - val_loss: 1494.5178
Epoch 5/100
73/73 - 0s - 3ms/step - loss: 1419.6442 - val_loss: 1439.6102
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1360.3734 - val_loss: 1375.8411
Epoch 7/100
73/73 - 0s - 3ms/step - loss: 1291.7279 - val_loss: 1303.6860
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1215.9290 - val_loss: 1225.1420
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1133.1494 - val_loss: 1140.4690
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 1047.5800 - val_loss: 1054.4409
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 958.8066 - val_loss: 966.1600
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 871.6562 - val_loss: 880.1720
Epoch 13/100
73/73 - 1s - 7ms/step - loss: 787.3809 - val_loss: 799.0370
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 707.8040

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1595.1993 - val_loss: 1460.6837
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1531.2174 - val_loss: 1397.8422
Epoch 3/100
73/73 - 0s - 3ms/step - loss: 1468.2932 - val_loss: 1335.0443
Epoch 4/100
73/73 - 0s - 2ms/step - loss: 1403.5804 - val_loss: 1268.5321
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1335.2002 - val_loss: 1200.0607
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1263.5477 - val_loss: 1127.7581
Epoch 7/100
73/73 - 0s - 3ms/step - loss: 1189.7480 - val_loss: 1054.6304
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1113.0509 - val_loss: 980.7320
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1034.5793 - val_loss: 905.0717
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 957.0483 - val_loss: 833.5278
Epoch 11/100
73/73 - 0s - 2ms/step - loss: 880.6580 - val_loss: 762.3588
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 808.0679 - val_loss: 696.2657
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 738.4482 - val_loss: 634.6333
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 672.8392 - v

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1593.3967 - val_loss: 1672.8541
Epoch 2/100
73/73 - 0s - 7ms/step - loss: 1554.2346 - val_loss: 1637.5645
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1522.5376 - val_loss: 1607.4194
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1494.5306 - val_loss: 1579.9882
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1466.7520 - val_loss: 1550.6965
Epoch 6/100
73/73 - 0s - 5ms/step - loss: 1435.0089 - val_loss: 1514.1459
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1393.5562 - val_loss: 1464.1478
Epoch 8/100
73/73 - 0s - 3ms/step - loss: 1339.0731 - val_loss: 1400.3361
Epoch 9/100
73/73 - 0s - 3ms/step - loss: 1271.9371 - val_loss: 1324.5204
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 1194.5364 - val_loss: 1238.4065
Epoch 11/100
73/73 - 0s - 3ms/step - loss: 1107.8225 - val_loss: 1144.8655
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 1016.2645 - val_loss: 1049.7395
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 923.9579 - val_loss: 953.9747
Epoch 14/100
73/73 - 0s - 2ms/step - loss: 832.

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1558.5383 - val_loss: 1604.3099
Epoch 2/100
73/73 - 0s - 2ms/step - loss: 1516.6239 - val_loss: 1559.2875
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1472.0751 - val_loss: 1509.9259
Epoch 4/100
73/73 - 0s - 5ms/step - loss: 1422.3862 - val_loss: 1452.7703
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1364.4750 - val_loss: 1387.8481
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1297.2684 - val_loss: 1313.0657
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1223.3169 - val_loss: 1231.5637
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1142.6738 - val_loss: 1144.3289
Epoch 9/100
73/73 - 0s - 5ms/step - loss: 1059.6898 - val_loss: 1056.3478
Epoch 10/100
73/73 - 0s - 3ms/step - loss: 973.8121 - val_loss: 964.9669
Epoch 11/100
73/73 - 0s - 5ms/step - loss: 885.6828 - val_loss: 873.0096
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 798.8553 - val_loss: 782.0267
Epoch 13/100
73/73 - 0s - 5ms/step - loss: 714.0811 - val_loss: 694.6373
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 635.0146 -

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1543.7438 - val_loss: 1459.5327
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1496.7909 - val_loss: 1412.1301
Epoch 3/100
73/73 - 0s - 3ms/step - loss: 1444.6631 - val_loss: 1359.2593
Epoch 4/100
73/73 - 0s - 5ms/step - loss: 1385.8685 - val_loss: 1300.4089
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1319.5713 - val_loss: 1232.9271
Epoch 6/100
73/73 - 0s - 3ms/step - loss: 1246.2181 - val_loss: 1159.0198
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1165.6078 - val_loss: 1077.1727
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1076.5088 - val_loss: 989.6047
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 980.4780 - val_loss: 895.9253
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 882.4080 - val_loss: 801.7293
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 786.0569 - val_loss: 712.3038
Epoch 12/100
73/73 - 0s - 3ms/step - loss: 693.7381 - val_loss: 626.2066
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 606.9291 - val_loss: 544.2479
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 528.5603 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1518.7998 - val_loss: 1530.7469
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1464.6064 - val_loss: 1470.9008
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1402.6045 - val_loss: 1401.6974
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1329.2277 - val_loss: 1321.9781
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1245.0718 - val_loss: 1230.6853
Epoch 6/100
73/73 - 0s - 2ms/step - loss: 1153.4958 - val_loss: 1134.0840
Epoch 7/100
73/73 - 0s - 2ms/step - loss: 1054.6530 - val_loss: 1031.0980
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 952.1093 - val_loss: 926.4475
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 850.5438 - val_loss: 826.0747
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 752.9775 - val_loss: 729.1204
Epoch 11/100
73/73 - 0s - 3ms/step - loss: 659.4266 - val_loss: 638.2475
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 574.7173 - val_loss: 557.6846
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 499.3230 - val_loss: 485.8298
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 433.2022 - val

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1546.8242 - val_loss: 1526.6401
Epoch 2/100
73/73 - 0s - 2ms/step - loss: 1497.3824 - val_loss: 1477.7952
Epoch 3/100
73/73 - 0s - 5ms/step - loss: 1445.0090 - val_loss: 1425.1598
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1388.5745 - val_loss: 1368.0292
Epoch 5/100
73/73 - 0s - 2ms/step - loss: 1326.0492 - val_loss: 1305.1879
Epoch 6/100
73/73 - 0s - 5ms/step - loss: 1257.4510 - val_loss: 1234.7433
Epoch 7/100
73/73 - 0s - 3ms/step - loss: 1181.9359 - val_loss: 1157.5077
Epoch 8/100
73/73 - 0s - 3ms/step - loss: 1098.0833 - val_loss: 1072.9525
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1008.1855 - val_loss: 983.1201
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 917.4464 - val_loss: 893.7610
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 824.4214 - val_loss: 805.1930
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 736.2304 - val_loss: 720.3030
Epoch 13/100
73/73 - 0s - 5ms/step - loss: 655.6074 - val_loss: 643.9536
Epoch 14/100
73/73 - 0s - 5ms/step - loss: 581.1805 - 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1528.0713 - val_loss: 1619.7882
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1481.2534 - val_loss: 1571.0906
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1433.3129 - val_loss: 1519.9734
Epoch 4/100
73/73 - 0s - 5ms/step - loss: 1383.0381 - val_loss: 1463.0248
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1327.2346 - val_loss: 1400.8768
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1265.8961 - val_loss: 1331.4935
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1198.8813 - val_loss: 1256.3562
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1126.6410 - val_loss: 1176.0192
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1048.6038 - val_loss: 1086.4178
Epoch 10/100
73/73 - 0s - 5ms/step - loss: 964.5452 - val_loss: 993.1689
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 875.9895 - val_loss: 893.9020
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 785.4948 - val_loss: 795.0612
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 695.9866 - val_loss: 697.1411
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 610.2405 -

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1601.6881 - val_loss: 1532.8838
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1556.6146 - val_loss: 1490.5651
Epoch 3/100
73/73 - 0s - 3ms/step - loss: 1514.4274 - val_loss: 1447.8835
Epoch 4/100
73/73 - 0s - 3ms/step - loss: 1470.0104 - val_loss: 1400.4514
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1419.5387 - val_loss: 1345.5154
Epoch 6/100
73/73 - 0s - 2ms/step - loss: 1361.6027 - val_loss: 1284.2109
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1296.6460 - val_loss: 1216.1066
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1225.7914 - val_loss: 1141.2299
Epoch 9/100
73/73 - 0s - 3ms/step - loss: 1149.0146 - val_loss: 1061.7080
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 1068.5797 - val_loss: 980.9888
Epoch 11/100
73/73 - 0s - 5ms/step - loss: 986.5965 - val_loss: 896.0951
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 901.5291 - val_loss: 809.9598
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 812.9539 - val_loss: 718.7625
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 721.3185 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1521.5618 - val_loss: 1470.3411
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1468.0305 - val_loss: 1413.3258
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1404.6349 - val_loss: 1346.5304
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1331.0533 - val_loss: 1270.2463
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1248.3381 - val_loss: 1185.2799
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1156.5525 - val_loss: 1092.1968
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1058.6948 - val_loss: 994.2595
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 956.1121 - val_loss: 891.8453
Epoch 9/100
73/73 - 0s - 5ms/step - loss: 854.5027 - val_loss: 794.8519
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 755.2376 - val_loss: 698.5416
Epoch 11/100
73/73 - 0s - 5ms/step - loss: 660.5739 - val_loss: 610.1022
Epoch 12/100
73/73 - 1s - 8ms/step - loss: 577.8886 - val_loss: 533.2976
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 503.7751 - val_loss: 463.7135
Epoch 14/100
73/73 - 0s - 5ms/step - loss: 439.3902 - val_

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1579.7340 - val_loss: 1561.8645
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1538.6068 - val_loss: 1521.1418
Epoch 3/100
73/73 - 0s - 3ms/step - loss: 1495.6064 - val_loss: 1477.7498
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1447.4122 - val_loss: 1429.3010
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1393.5552 - val_loss: 1375.6796
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1332.4370 - val_loss: 1315.4113
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1266.9313 - val_loss: 1251.8936
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1197.2477 - val_loss: 1184.2052
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1124.2697 - val_loss: 1112.2046
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 1047.9425 - val_loss: 1039.5554
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 970.8114 - val_loss: 965.5494
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 894.0180 - val_loss: 891.7092
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 817.6562 - val_loss: 819.0550
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 745.9561

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 16ms/step - loss: 1595.0913 - val_loss: 1537.7408
Epoch 2/100
73/73 - 0s - 7ms/step - loss: 1553.9424 - val_loss: 1499.1252
Epoch 3/100
73/73 - 1s - 8ms/step - loss: 1514.1832 - val_loss: 1458.5938
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1470.0673 - val_loss: 1412.4934
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1418.9739 - val_loss: 1358.8641
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1357.1771 - val_loss: 1294.7878
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1286.5216 - val_loss: 1222.7780
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1206.2435 - val_loss: 1141.7915
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1118.5026 - val_loss: 1055.1841
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 1024.5277 - val_loss: 960.9814
Epoch 11/100
73/73 - 0s - 3ms/step - loss: 925.0126 - val_loss: 865.7106
Epoch 12/100
73/73 - 0s - 2ms/step - loss: 826.0998 - val_loss: 774.1140
Epoch 13/100
73/73 - 0s - 3ms/step - loss: 732.8484 - val_loss: 686.1633
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 647.8734 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1544.8367 - val_loss: 1519.4885
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1499.7654 - val_loss: 1471.0205
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1446.1125 - val_loss: 1411.8776
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1383.1930 - val_loss: 1343.2461
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1307.7866 - val_loss: 1261.8075
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1223.1655 - val_loss: 1171.4362
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1129.8707 - val_loss: 1073.7738
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1029.8728 - val_loss: 971.4747
Epoch 9/100
73/73 - 1s - 8ms/step - loss: 929.5520 - val_loss: 871.6263
Epoch 10/100
73/73 - 0s - 3ms/step - loss: 829.2073 - val_loss: 771.5168
Epoch 11/100
73/73 - 0s - 2ms/step - loss: 729.2354 - val_loss: 676.0500
Epoch 12/100
73/73 - 0s - 3ms/step - loss: 633.9888 - val_loss: 586.3567
Epoch 13/100
73/73 - 0s - 2ms/step - loss: 548.2275 - val_loss: 507.5456
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 472.6597 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 5s - 63ms/step - loss: 1530.0647 - val_loss: 1573.9535
Epoch 2/100
73/73 - 0s - 5ms/step - loss: 1478.2637 - val_loss: 1519.9403
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1425.4314 - val_loss: 1462.6434
Epoch 4/100
73/73 - 0s - 5ms/step - loss: 1368.5409 - val_loss: 1401.2097
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1306.8993 - val_loss: 1333.3923
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1238.9211 - val_loss: 1259.9708
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1165.2349 - val_loss: 1181.2889
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1086.4523 - val_loss: 1097.5568
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1005.4915 - val_loss: 1011.8728
Epoch 10/100
73/73 - 0s - 3ms/step - loss: 921.9623 - val_loss: 926.9332
Epoch 11/100
73/73 - 0s - 3ms/step - loss: 840.5561 - val_loss: 841.9047
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 761.6114 - val_loss: 761.7534
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 686.9089 - val_loss: 685.8345
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 616.7797 -

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1495.9117 - val_loss: 1548.2428
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1448.4833 - val_loss: 1499.9324
Epoch 3/100
73/73 - 0s - 2ms/step - loss: 1395.0863 - val_loss: 1444.9622
Epoch 4/100
73/73 - 0s - 2ms/step - loss: 1333.1146 - val_loss: 1379.9573
Epoch 5/100
73/73 - 0s - 3ms/step - loss: 1261.2889 - val_loss: 1305.8796
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1179.6356 - val_loss: 1223.1350
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1091.9829 - val_loss: 1134.7389
Epoch 8/100
73/73 - 0s - 3ms/step - loss: 999.4738 - val_loss: 1042.2780
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 903.5795 - val_loss: 946.0757
Epoch 10/100
73/73 - 0s - 3ms/step - loss: 805.4555 - val_loss: 848.0540
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 708.5204 - val_loss: 752.3208
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 615.3644 - val_loss: 657.9038
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 530.4944 - val_loss: 576.4484
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 456.3484 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1534.9926 - val_loss: 1503.0905
Epoch 2/100
73/73 - 0s - 7ms/step - loss: 1493.4108 - val_loss: 1453.4714
Epoch 3/100
73/73 - 0s - 5ms/step - loss: 1441.5166 - val_loss: 1391.4779
Epoch 4/100
73/73 - 0s - 3ms/step - loss: 1378.0181 - val_loss: 1319.5310
Epoch 5/100
73/73 - 0s - 3ms/step - loss: 1307.4296 - val_loss: 1238.5496
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1227.2637 - val_loss: 1151.5214
Epoch 7/100
73/73 - 0s - 2ms/step - loss: 1137.6075 - val_loss: 1052.8827
Epoch 8/100
73/73 - 0s - 3ms/step - loss: 1041.1472 - val_loss: 949.6347
Epoch 9/100
73/73 - 0s - 2ms/step - loss: 940.7151 - val_loss: 845.1027
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 840.9875 - val_loss: 745.0120
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 743.3133 - val_loss: 650.3040
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 650.8652 - val_loss: 563.6335
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 569.2830 - val_loss: 489.9279
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 496.6050 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1595.5417 - val_loss: 1502.4677
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1558.5907 - val_loss: 1468.9175
Epoch 3/100
73/73 - 0s - 5ms/step - loss: 1524.4175 - val_loss: 1434.8693
Epoch 4/100
73/73 - 0s - 2ms/step - loss: 1487.8425 - val_loss: 1395.8260
Epoch 5/100
73/73 - 0s - 5ms/step - loss: 1445.4640 - val_loss: 1350.5514
Epoch 6/100
73/73 - 0s - 3ms/step - loss: 1395.9658 - val_loss: 1297.7333
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1339.1176 - val_loss: 1235.2290
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1271.4257 - val_loss: 1163.6837
Epoch 9/100
73/73 - 0s - 2ms/step - loss: 1193.9677 - val_loss: 1082.4757
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 1107.4327 - val_loss: 995.0645
Epoch 11/100
73/73 - 0s - 3ms/step - loss: 1014.9506 - val_loss: 902.4304
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 918.3707 - val_loss: 809.1440
Epoch 13/100
73/73 - 0s - 3ms/step - loss: 821.7607 - val_loss: 718.0919
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 728.4348

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1551.2792 - val_loss: 1491.6345
Epoch 2/100
73/73 - 0s - 7ms/step - loss: 1502.2292 - val_loss: 1442.2124
Epoch 3/100
73/73 - 0s - 2ms/step - loss: 1447.9219 - val_loss: 1386.6506
Epoch 4/100
73/73 - 0s - 5ms/step - loss: 1386.5881 - val_loss: 1323.6506
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1316.9973 - val_loss: 1253.9530
Epoch 6/100
73/73 - 0s - 2ms/step - loss: 1240.1637 - val_loss: 1177.6506
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1155.4209 - val_loss: 1097.0656
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1067.1610 - val_loss: 1010.4857
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 974.8436 - val_loss: 922.1797
Epoch 10/100
73/73 - 0s - 5ms/step - loss: 883.5235 - val_loss: 835.2175
Epoch 11/100
73/73 - 0s - 5ms/step - loss: 792.4965 - val_loss: 748.8775
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 703.8141 - val_loss: 664.8596
Epoch 13/100
73/73 - 0s - 5ms/step - loss: 619.7989 - val_loss: 587.2509
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 541.7466 - v

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1516.8184 - val_loss: 1530.5773
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1471.2821 - val_loss: 1484.0625
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1421.8021 - val_loss: 1430.9087
Epoch 4/100
73/73 - 0s - 3ms/step - loss: 1364.1729 - val_loss: 1369.0085
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1296.3883 - val_loss: 1295.3921
Epoch 6/100
73/73 - 0s - 2ms/step - loss: 1218.7582 - val_loss: 1214.7537
Epoch 7/100
73/73 - 0s - 6ms/step - loss: 1133.5768 - val_loss: 1128.4537
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1045.0824 - val_loss: 1039.3966
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 954.8348 - val_loss: 948.1002
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 865.9443 - val_loss: 861.7395
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 779.0229 - val_loss: 776.5821
Epoch 12/100
73/73 - 0s - 3ms/step - loss: 696.0763 - val_loss: 697.1167
Epoch 13/100
73/73 - 0s - 5ms/step - loss: 618.2619 - val_loss: 622.7813
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 545.7326 - v

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1586.9326 - val_loss: 1464.2706
Epoch 2/100
73/73 - 0s - 7ms/step - loss: 1533.7821 - val_loss: 1415.9332
Epoch 3/100
73/73 - 0s - 5ms/step - loss: 1482.3800 - val_loss: 1366.3821
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1429.9939 - val_loss: 1314.6827
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1374.8193 - val_loss: 1259.6868
Epoch 6/100
73/73 - 1s - 8ms/step - loss: 1315.6550 - val_loss: 1200.2168
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1252.6964 - val_loss: 1136.8344
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1185.9185 - val_loss: 1071.5872
Epoch 9/100
73/73 - 0s - 3ms/step - loss: 1116.2870 - val_loss: 1003.8508
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 1044.9657 - val_loss: 934.7499
Epoch 11/100
73/73 - 0s - 3ms/step - loss: 972.4268 - val_loss: 864.8128
Epoch 12/100
73/73 - 0s - 5ms/step - loss: 897.3873 - val_loss: 795.5458
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 821.7094 - val_loss: 723.8560
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 745.8312 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 23ms/step - loss: 1515.7062 - val_loss: 1560.4777
Epoch 2/100
73/73 - 0s - 5ms/step - loss: 1471.2278 - val_loss: 1512.9161
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1422.7897 - val_loss: 1459.5785
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1366.9888 - val_loss: 1397.9391
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1305.3253 - val_loss: 1329.1165
Epoch 6/100
73/73 - 0s - 3ms/step - loss: 1236.7158 - val_loss: 1254.4921
Epoch 7/100
73/73 - 0s - 2ms/step - loss: 1161.2075 - val_loss: 1173.9457
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1081.4487 - val_loss: 1089.0305
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 999.8531 - val_loss: 1000.8185
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 916.4346 - val_loss: 915.5893
Epoch 11/100
73/73 - 0s - 5ms/step - loss: 835.2064 - val_loss: 830.4369
Epoch 12/100
73/73 - 0s - 3ms/step - loss: 756.1900 - val_loss: 751.0769
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 681.2474 - val_loss: 675.2065
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 611.4540 - 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1542.4283 - val_loss: 1531.1899
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1489.0094 - val_loss: 1479.3414
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1433.6406 - val_loss: 1423.8187
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1374.8289 - val_loss: 1364.6652
Epoch 5/100
73/73 - 0s - 3ms/step - loss: 1311.8737 - val_loss: 1300.4927
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1243.3225 - val_loss: 1231.5929
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1172.0358 - val_loss: 1159.7983
Epoch 8/100
73/73 - 0s - 5ms/step - loss: 1096.4445 - val_loss: 1081.8705
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1016.5880 - val_loss: 999.1005
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 931.3768 - val_loss: 911.8375
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 843.1846 - val_loss: 821.9394
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 753.0259 - val_loss: 730.9194
Epoch 13/100
73/73 - 0s - 3ms/step - loss: 666.4303 - val_loss: 645.2002
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 583.3672 - 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 19ms/step - loss: 1595.0759 - val_loss: 1561.5127
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1547.6877 - val_loss: 1517.1377
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1505.4983 - val_loss: 1476.2048
Epoch 4/100
73/73 - 0s - 3ms/step - loss: 1465.8728 - val_loss: 1436.6725
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1426.2791 - val_loss: 1396.1367
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1384.7136 - val_loss: 1354.3140
Epoch 7/100
73/73 - 0s - 3ms/step - loss: 1343.0740 - val_loss: 1311.7184
Epoch 8/100
73/73 - 0s - 3ms/step - loss: 1299.3723 - val_loss: 1266.9983
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1253.3927 - val_loss: 1220.6017
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 1205.2235 - val_loss: 1171.4796
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 1155.9913 - val_loss: 1120.6427
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 1105.0482 - val_loss: 1069.5337
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 1052.9956 - val_loss: 1016.0743
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 10

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1531.3671 - val_loss: 1442.4904
Epoch 2/100
73/73 - 0s - 7ms/step - loss: 1476.3079 - val_loss: 1383.2010
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1414.9766 - val_loss: 1319.5648
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1347.9215 - val_loss: 1248.5724
Epoch 5/100
73/73 - 0s - 3ms/step - loss: 1274.1228 - val_loss: 1172.7238
Epoch 6/100
73/73 - 0s - 3ms/step - loss: 1194.3965 - val_loss: 1092.4373
Epoch 7/100
73/73 - 0s - 3ms/step - loss: 1108.8065 - val_loss: 1008.1767
Epoch 8/100
73/73 - 0s - 2ms/step - loss: 1020.3167 - val_loss: 921.7271
Epoch 9/100
73/73 - 0s - 5ms/step - loss: 930.3719 - val_loss: 836.9966
Epoch 10/100
73/73 - 0s - 2ms/step - loss: 841.5123 - val_loss: 752.8841
Epoch 11/100
73/73 - 0s - 5ms/step - loss: 753.9167 - val_loss: 670.1508
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 668.3142 - val_loss: 590.4070
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 586.1187 - val_loss: 516.6331
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 508.7891 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1568.8186 - val_loss: 1448.7322
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1524.2456 - val_loss: 1402.4482
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1475.3679 - val_loss: 1350.2434
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1420.5443 - val_loss: 1294.0233
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1360.0160 - val_loss: 1231.1111
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1293.1995 - val_loss: 1164.6141
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1221.0533 - val_loss: 1092.4849
Epoch 8/100
73/73 - 0s - 3ms/step - loss: 1143.3695 - val_loss: 1017.0353
Epoch 9/100
73/73 - 0s - 2ms/step - loss: 1062.5370 - val_loss: 940.5482
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 980.6968 - val_loss: 863.5212
Epoch 11/100
73/73 - 0s - 5ms/step - loss: 899.3176 - val_loss: 788.6296
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 819.5137 - val_loss: 715.2705
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 742.0961 - val_loss: 647.2722
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 669.3939 - 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1563.5161 - val_loss: 1536.9769
Epoch 2/100
73/73 - 0s - 2ms/step - loss: 1524.2311 - val_loss: 1497.3766
Epoch 3/100
73/73 - 0s - 5ms/step - loss: 1478.2375 - val_loss: 1450.4325
Epoch 4/100
73/73 - 0s - 3ms/step - loss: 1422.0007 - val_loss: 1393.3959
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1354.4287 - val_loss: 1327.0272
Epoch 6/100
73/73 - 0s - 2ms/step - loss: 1276.6199 - val_loss: 1250.2891
Epoch 7/100
73/73 - 0s - 2ms/step - loss: 1190.7653 - val_loss: 1167.6927
Epoch 8/100
73/73 - 0s - 5ms/step - loss: 1098.1478 - val_loss: 1079.5739
Epoch 9/100
73/73 - 0s - 2ms/step - loss: 1005.3531 - val_loss: 990.6016
Epoch 10/100
73/73 - 0s - 2ms/step - loss: 912.7349 - val_loss: 900.8013
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 822.3832 - val_loss: 811.3266
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 734.8917 - val_loss: 723.3632
Epoch 13/100
73/73 - 0s - 3ms/step - loss: 650.7064 - val_loss: 636.5162
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 571.9056 - 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1553.4922 - val_loss: 1443.6218
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1505.1383 - val_loss: 1394.9741
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1454.2405 - val_loss: 1343.1326
Epoch 4/100
73/73 - 0s - 2ms/step - loss: 1398.9114 - val_loss: 1286.8757
Epoch 5/100
73/73 - 0s - 5ms/step - loss: 1336.6735 - val_loss: 1224.6023
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1266.1969 - val_loss: 1154.9565
Epoch 7/100
73/73 - 0s - 2ms/step - loss: 1186.7554 - val_loss: 1078.4747
Epoch 8/100
73/73 - 0s - 3ms/step - loss: 1100.3198 - val_loss: 995.3208
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 1009.7208 - val_loss: 910.1833
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 917.0650 - val_loss: 822.3085
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 823.5687 - val_loss: 737.9088
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 732.6961 - val_loss: 656.1888
Epoch 13/100
73/73 - 0s - 3ms/step - loss: 647.2189 - val_loss: 579.6157
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 568.5473 - v

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1468.3555 - val_loss: 1497.6931
Epoch 2/100
73/73 - 1s - 7ms/step - loss: 1405.6725 - val_loss: 1434.8069
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1337.4113 - val_loss: 1367.0874
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1263.7808 - val_loss: 1292.3430
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1183.0564 - val_loss: 1210.4781
Epoch 6/100
73/73 - 0s - 3ms/step - loss: 1096.8280 - val_loss: 1125.2954
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1006.9739 - val_loss: 1034.1670
Epoch 8/100
73/73 - 0s - 3ms/step - loss: 915.0902 - val_loss: 941.1835
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 823.2802 - val_loss: 848.3824
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 735.3856 - val_loss: 760.0955
Epoch 11/100
73/73 - 0s - 2ms/step - loss: 652.6036 - val_loss: 675.9658
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 576.2217 - val_loss: 596.7675
Epoch 13/100
73/73 - 0s - 5ms/step - loss: 508.0035 - val_loss: 527.2968
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 448.5965 - val

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1571.3655 - val_loss: 1522.6897
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1527.5459 - val_loss: 1478.7509
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1481.4413 - val_loss: 1431.5275
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1431.6815 - val_loss: 1379.5742
Epoch 5/100
73/73 - 0s - 5ms/step - loss: 1377.0464 - val_loss: 1322.4988
Epoch 6/100
73/73 - 0s - 3ms/step - loss: 1316.2740 - val_loss: 1258.8750
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1249.5184 - val_loss: 1189.4105
Epoch 8/100
73/73 - 0s - 5ms/step - loss: 1176.8088 - val_loss: 1114.0819
Epoch 9/100
73/73 - 0s - 2ms/step - loss: 1098.6660 - val_loss: 1034.8590
Epoch 10/100
73/73 - 0s - 5ms/step - loss: 1015.5462 - val_loss: 950.7398
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 929.5720 - val_loss: 866.3074
Epoch 12/100
73/73 - 0s - 2ms/step - loss: 842.9973 - val_loss: 779.8226
Epoch 13/100
73/73 - 0s - 2ms/step - loss: 755.7776 - val_loss: 695.5055
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 670.2732 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1566.1045 - val_loss: 1529.3691
Epoch 2/100
73/73 - 1s - 7ms/step - loss: 1517.7318 - val_loss: 1479.9408
Epoch 3/100
73/73 - 0s - 3ms/step - loss: 1466.2540 - val_loss: 1426.1382
Epoch 4/100
73/73 - 0s - 3ms/step - loss: 1409.3241 - val_loss: 1367.0236
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1345.9943 - val_loss: 1303.1752
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1277.8811 - val_loss: 1233.9493
Epoch 7/100
73/73 - 0s - 3ms/step - loss: 1203.7378 - val_loss: 1159.3533
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1125.6744 - val_loss: 1080.9642
Epoch 9/100
73/73 - 0s - 3ms/step - loss: 1044.3586 - val_loss: 999.2689
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 959.7311 - val_loss: 915.2776
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 873.4868 - val_loss: 830.4761
Epoch 12/100
73/73 - 0s - 3ms/step - loss: 788.5046 - val_loss: 747.3482
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 704.7169 - val_loss: 667.9896
Epoch 14/100
73/73 - 0s - 5ms/step - loss: 627.0171 - 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1553.8796 - val_loss: 1381.6793
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1510.4000 - val_loss: 1338.0881
Epoch 3/100
73/73 - 0s - 3ms/step - loss: 1459.9133 - val_loss: 1287.2689
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1399.2249 - val_loss: 1225.5016
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1324.5435 - val_loss: 1148.3569
Epoch 6/100
73/73 - 0s - 3ms/step - loss: 1235.2260 - val_loss: 1059.0715
Epoch 7/100
73/73 - 0s - 2ms/step - loss: 1131.6097 - val_loss: 959.2386
Epoch 8/100
73/73 - 0s - 2ms/step - loss: 1018.3315 - val_loss: 854.0968
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 903.0099 - val_loss: 750.7003
Epoch 10/100
73/73 - 0s - 2ms/step - loss: 789.0161 - val_loss: 653.8291
Epoch 11/100
73/73 - 0s - 2ms/step - loss: 681.1479 - val_loss: 562.8909
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 584.9902 - val_loss: 485.1413
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 499.1715 - val_loss: 419.4119
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 428.3701 - val

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 18ms/step - loss: 1542.1595 - val_loss: 1477.1036
Epoch 2/100
73/73 - 0s - 2ms/step - loss: 1493.0663 - val_loss: 1426.4125
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1440.3264 - val_loss: 1369.9707
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1381.1451 - val_loss: 1307.2119
Epoch 5/100
73/73 - 0s - 2ms/step - loss: 1314.4956 - val_loss: 1235.2837
Epoch 6/100
73/73 - 0s - 2ms/step - loss: 1238.2083 - val_loss: 1156.5264
Epoch 7/100
73/73 - 0s - 3ms/step - loss: 1159.2573 - val_loss: 1074.1879
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1074.0076 - val_loss: 987.9657
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 984.1102 - val_loss: 896.7674
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 890.9879 - val_loss: 804.3149
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 798.2251 - val_loss: 716.2579
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 708.2137 - val_loss: 633.6020
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 622.5389 - val_loss: 555.6218
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 546.0651 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 18ms/step - loss: 1512.0609 - val_loss: 1413.0547
Epoch 2/100
73/73 - 0s - 2ms/step - loss: 1462.9929 - val_loss: 1361.4561
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1408.6160 - val_loss: 1304.8250
Epoch 4/100
73/73 - 0s - 3ms/step - loss: 1347.8658 - val_loss: 1242.3781
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1281.5110 - val_loss: 1174.0167
Epoch 6/100
73/73 - 0s - 3ms/step - loss: 1208.8479 - val_loss: 1100.0221
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1128.3743 - val_loss: 1020.7660
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1043.3623 - val_loss: 934.9169
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 953.1871 - val_loss: 845.8430
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 859.2719 - val_loss: 755.2718
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 766.7300 - val_loss: 666.9556
Epoch 12/100
73/73 - 0s - 3ms/step - loss: 676.6155 - val_loss: 582.6723
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 589.8627 - val_loss: 503.9048
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 510.2402 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 18ms/step - loss: 1506.2241 - val_loss: 1501.7573
Epoch 2/100
73/73 - 0s - 2ms/step - loss: 1458.0979 - val_loss: 1452.4708
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1405.5609 - val_loss: 1398.6797
Epoch 4/100
73/73 - 0s - 2ms/step - loss: 1347.6729 - val_loss: 1336.2246
Epoch 5/100
73/73 - 0s - 3ms/step - loss: 1280.6420 - val_loss: 1264.8695
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1205.9089 - val_loss: 1184.1862
Epoch 7/100
73/73 - 0s - 3ms/step - loss: 1121.0315 - val_loss: 1093.2025
Epoch 8/100
73/73 - 0s - 2ms/step - loss: 1029.9979 - val_loss: 998.8359
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 936.5181 - val_loss: 902.8638
Epoch 10/100
73/73 - 0s - 3ms/step - loss: 844.3451 - val_loss: 807.5461
Epoch 11/100
73/73 - 0s - 3ms/step - loss: 755.4235 - val_loss: 717.2652
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 673.5796 - val_loss: 635.2308
Epoch 13/100
73/73 - 0s - 3ms/step - loss: 601.0316 - val_loss: 562.2897
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 536.6376 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1515.2872 - val_loss: 1505.3562
Epoch 2/100
73/73 - 0s - 5ms/step - loss: 1469.9790 - val_loss: 1460.1381
Epoch 3/100
73/73 - 0s - 3ms/step - loss: 1417.6844 - val_loss: 1407.3527
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1356.1392 - val_loss: 1343.4832
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1283.6891 - val_loss: 1269.8101
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1200.5693 - val_loss: 1185.2190
Epoch 7/100
73/73 - 0s - 5ms/step - loss: 1106.0366 - val_loss: 1091.6674
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1002.1755 - val_loss: 987.6784
Epoch 9/100
73/73 - 0s - 2ms/step - loss: 890.4352 - val_loss: 880.0733
Epoch 10/100
73/73 - 0s - 3ms/step - loss: 778.1392 - val_loss: 773.3229
Epoch 11/100
73/73 - 0s - 2ms/step - loss: 673.4684 - val_loss: 674.9273
Epoch 12/100
73/73 - 0s - 3ms/step - loss: 577.9470 - val_loss: 584.2484
Epoch 13/100
73/73 - 0s - 3ms/step - loss: 495.6133 - val_loss: 507.1660
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 426.5200 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 18ms/step - loss: 1536.1521 - val_loss: 1485.4729
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1487.1836 - val_loss: 1437.9697
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1434.6206 - val_loss: 1386.1877
Epoch 4/100
73/73 - 0s - 3ms/step - loss: 1376.2812 - val_loss: 1327.1194
Epoch 5/100
73/73 - 0s - 2ms/step - loss: 1308.8809 - val_loss: 1259.1361
Epoch 6/100
73/73 - 0s - 3ms/step - loss: 1234.4326 - val_loss: 1185.0535
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1154.0558 - val_loss: 1106.4688
Epoch 8/100
73/73 - 0s - 2ms/step - loss: 1067.7982 - val_loss: 1024.7111
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 979.9783 - val_loss: 940.6011
Epoch 10/100
73/73 - 0s - 5ms/step - loss: 891.9874 - val_loss: 857.7266
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 804.8248 - val_loss: 775.9855
Epoch 12/100
73/73 - 0s - 3ms/step - loss: 719.5246 - val_loss: 695.5722
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 639.4037 - val_loss: 619.0643
Epoch 14/100
73/73 - 0s - 2ms/step - loss: 563.8156 - v

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 19ms/step - loss: 1454.5312 - val_loss: 1594.5863
Epoch 2/100
73/73 - 0s - 4ms/step - loss: 1406.6129 - val_loss: 1534.2173
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1348.7810 - val_loss: 1461.5302
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1279.8019 - val_loss: 1376.4160
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1199.8490 - val_loss: 1279.9150
Epoch 6/100
73/73 - 0s - 3ms/step - loss: 1110.6522 - val_loss: 1173.9352
Epoch 7/100
73/73 - 0s - 3ms/step - loss: 1015.1386 - val_loss: 1063.6914
Epoch 8/100
73/73 - 0s - 3ms/step - loss: 916.3352 - val_loss: 952.4249
Epoch 9/100
73/73 - 0s - 2ms/step - loss: 817.6544 - val_loss: 844.2457
Epoch 10/100
73/73 - 0s - 3ms/step - loss: 723.5217 - val_loss: 741.5152
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 635.5406 - val_loss: 648.2213
Epoch 12/100
73/73 - 0s - 2ms/step - loss: 557.0286 - val_loss: 569.8277
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 486.4488 - val_loss: 499.5089
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 426.4488 - val

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1502.5137 - val_loss: 1582.2396
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1448.3237 - val_loss: 1521.0675
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1388.3429 - val_loss: 1453.6556
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1320.3652 - val_loss: 1378.2507
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1246.4308 - val_loss: 1295.3911
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1165.3474 - val_loss: 1205.3553
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1079.5149 - val_loss: 1111.2905
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 989.5961 - val_loss: 1014.4466
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 899.1900 - val_loss: 917.3177
Epoch 10/100
73/73 - 0s - 2ms/step - loss: 809.9415 - val_loss: 823.3782
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 724.4072 - val_loss: 733.8409
Epoch 12/100
73/73 - 0s - 2ms/step - loss: 643.8538 - val_loss: 648.8442
Epoch 13/100
73/73 - 0s - 3ms/step - loss: 569.6005 - val_loss: 572.6686
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 503.4047 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 16ms/step - loss: 1502.0354 - val_loss: 1494.7286
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1451.2117 - val_loss: 1442.7135
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1393.1646 - val_loss: 1382.3098
Epoch 4/100
73/73 - 0s - 5ms/step - loss: 1325.9214 - val_loss: 1312.8398
Epoch 5/100
73/73 - 1s - 7ms/step - loss: 1249.5698 - val_loss: 1235.4742
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1165.4092 - val_loss: 1148.4558
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1074.3124 - val_loss: 1057.2699
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 978.0860 - val_loss: 963.6125
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 884.4099 - val_loss: 870.7757
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 792.4142 - val_loss: 779.5012
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 704.2720 - val_loss: 692.3724
Epoch 12/100
73/73 - 0s - 3ms/step - loss: 621.9875 - val_loss: 612.6440
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 547.6783 - val_loss: 539.4246
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 481.2240 - val

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1566.6456 - val_loss: 1436.1248
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1527.6122 - val_loss: 1396.7191
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1481.2385 - val_loss: 1349.8314
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1424.1191 - val_loss: 1289.7430
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1350.2701 - val_loss: 1215.5348
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1262.5283 - val_loss: 1129.5267
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1161.3467 - val_loss: 1034.0410
Epoch 8/100
73/73 - 0s - 3ms/step - loss: 1051.4446 - val_loss: 933.3472
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 938.7151 - val_loss: 831.1819
Epoch 10/100
73/73 - 0s - 5ms/step - loss: 826.8717 - val_loss: 730.8672
Epoch 11/100
73/73 - 1s - 7ms/step - loss: 719.0750 - val_loss: 635.8832
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 618.6072 - val_loss: 547.0438
Epoch 13/100
73/73 - 0s - 2ms/step - loss: 525.1917 - val_loss: 467.0988
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 445.2419 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 17ms/step - loss: 1530.4967 - val_loss: 1482.3693
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1484.9885 - val_loss: 1436.6805
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1437.3462 - val_loss: 1387.4426
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1384.2854 - val_loss: 1333.0011
Epoch 5/100
73/73 - 0s - 3ms/step - loss: 1325.3392 - val_loss: 1271.1029
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1258.0046 - val_loss: 1199.8707
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1179.6941 - val_loss: 1117.0891
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1092.3550 - val_loss: 1025.6688
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 997.7493 - val_loss: 929.5219
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 899.5045 - val_loss: 831.3133
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 800.0782 - val_loss: 734.3148
Epoch 12/100
73/73 - 0s - 3ms/step - loss: 704.7901 - val_loss: 643.2114
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 615.7939 - val_loss: 560.3895
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 534.9843 - v

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 19ms/step - loss: 1556.3247 - val_loss: 1450.0190
Epoch 2/100
73/73 - 0s - 4ms/step - loss: 1504.5890 - val_loss: 1399.1265
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1447.0665 - val_loss: 1341.9187
Epoch 4/100
73/73 - 0s - 5ms/step - loss: 1380.0852 - val_loss: 1275.4097
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1302.8318 - val_loss: 1198.9779
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1217.3585 - val_loss: 1115.2030
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1123.7701 - val_loss: 1025.3435
Epoch 8/100
73/73 - 0s - 3ms/step - loss: 1026.7538 - val_loss: 933.6212
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 927.4908 - val_loss: 840.2229
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 830.7045 - val_loss: 749.6733
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 736.9761 - val_loss: 664.5611
Epoch 12/100
73/73 - 0s - 5ms/step - loss: 651.0772 - val_loss: 586.1533
Epoch 13/100
73/73 - 0s - 2ms/step - loss: 572.1746 - val_loss: 515.4530
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 501.4315 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 21ms/step - loss: 1591.7158 - val_loss: 1499.2480
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1540.2196 - val_loss: 1451.5258
Epoch 3/100
73/73 - 0s - 3ms/step - loss: 1487.0863 - val_loss: 1401.4946
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1430.3440 - val_loss: 1347.8042
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1369.3539 - val_loss: 1288.7339
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1302.3787 - val_loss: 1226.4911
Epoch 7/100
73/73 - 0s - 3ms/step - loss: 1230.1692 - val_loss: 1158.4878
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1154.0710 - val_loss: 1087.3784
Epoch 9/100
73/73 - 0s - 2ms/step - loss: 1074.0320 - val_loss: 1012.2042
Epoch 10/100
73/73 - 0s - 5ms/step - loss: 993.9295 - val_loss: 939.0482
Epoch 11/100
73/73 - 0s - 3ms/step - loss: 913.6926 - val_loss: 864.7568
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 833.5249 - val_loss: 792.4651
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 755.7234 - val_loss: 722.1021
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 681.8821 -

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1486.5466 - val_loss: 1529.8715
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1430.1814 - val_loss: 1466.4275
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1370.2842 - val_loss: 1398.4719
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1308.5920 - val_loss: 1327.3887
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1241.5699 - val_loss: 1250.7466
Epoch 6/100
73/73 - 0s - 4ms/step - loss: 1168.3973 - val_loss: 1167.3311
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1090.8289 - val_loss: 1080.5442
Epoch 8/100
73/73 - 0s - 3ms/step - loss: 1009.5876 - val_loss: 990.6487
Epoch 9/100
73/73 - 0s - 4ms/step - loss: 927.1650 - val_loss: 901.0008
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 844.8168 - val_loss: 811.9573
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 763.1945 - val_loss: 725.0336
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 682.6953 - val_loss: 641.9747
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 609.6381 - val_loss: 566.6298
Epoch 14/100
73/73 - 0s - 2ms/step - loss: 542.0135 - va

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 18ms/step - loss: 1499.2784 - val_loss: 1563.4905
Epoch 2/100
73/73 - 0s - 3ms/step - loss: 1458.9603 - val_loss: 1518.1936
Epoch 3/100
73/73 - 0s - 5ms/step - loss: 1410.7017 - val_loss: 1462.5046
Epoch 4/100
73/73 - 0s - 5ms/step - loss: 1351.7510 - val_loss: 1392.4866
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1279.9817 - val_loss: 1310.9108
Epoch 6/100
73/73 - 1s - 8ms/step - loss: 1196.0200 - val_loss: 1216.4509
Epoch 7/100
73/73 - 0s - 3ms/step - loss: 1103.5638 - val_loss: 1115.0354
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1005.1583 - val_loss: 1010.3430
Epoch 9/100
73/73 - 0s - 3ms/step - loss: 903.1598 - val_loss: 901.9070
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 801.8340 - val_loss: 795.5828
Epoch 11/100
73/73 - 0s - 3ms/step - loss: 702.7541 - val_loss: 695.3539
Epoch 12/100
73/73 - 0s - 4ms/step - loss: 611.9161 - val_loss: 603.6934
Epoch 13/100
73/73 - 0s - 4ms/step - loss: 529.2444 - val_loss: 521.5085
Epoch 14/100
73/73 - 0s - 3ms/step - loss: 457.1382 - v

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1558.3594 - val_loss: 1714.3328
Epoch 2/100
73/73 - 0s - 6ms/step - loss: 1515.2831 - val_loss: 1667.5834
Epoch 3/100
73/73 - 0s - 4ms/step - loss: 1477.0425 - val_loss: 1625.8562
Epoch 4/100
73/73 - 0s - 4ms/step - loss: 1441.5209 - val_loss: 1585.5920
Epoch 5/100
73/73 - 0s - 4ms/step - loss: 1407.5884 - val_loss: 1546.7296
Epoch 6/100
73/73 - 0s - 3ms/step - loss: 1373.0643 - val_loss: 1506.6562
Epoch 7/100
73/73 - 0s - 4ms/step - loss: 1337.1324 - val_loss: 1464.0582
Epoch 8/100
73/73 - 0s - 4ms/step - loss: 1297.0039 - val_loss: 1415.1262
Epoch 9/100
73/73 - 1s - 7ms/step - loss: 1247.0864 - val_loss: 1353.7610
Epoch 10/100
73/73 - 0s - 4ms/step - loss: 1187.8424 - val_loss: 1283.5240
Epoch 11/100
73/73 - 0s - 4ms/step - loss: 1120.4866 - val_loss: 1203.0083
Epoch 12/100
73/73 - 0s - 2ms/step - loss: 1046.7235 - val_loss: 1119.4590
Epoch 13/100
73/73 - 0s - 2ms/step - loss: 971.9941 - val_loss: 1033.9873
Epoch 14/100
73/73 - 0s - 4ms/step - loss: 896

In [34]:
# 4. Calculate mean and standard deviation of MSE
mean_mse = np.mean(mse_list)
std_mse = np.std(mse_list)

In [35]:
# 5. Print results after normalization with 100 epochs
print(f"Mean of MSE over 100 runs: {mean_mse}")
print(f"Standard Deviation of MSE over 100 runs: {std_mse}")

Mean of MSE over 100 runs: 77.90864362902043
Standard Deviation of MSE over 100 runs: 14.802098866276834


In [29]:
# Print results after normalization with 50 epochs
print(f"Mean of MSE over 50 runs: {mean_mse}")
print(f"Standard Deviation of MSE over 50 runs: {std_mse}")

Mean of MSE over 50 runs: 130.31662939683622
Standard Deviation of MSE over 50 runs: 11.836501926494584


# Mean MSE:

## The mean MSE for 50 epochs is 130.31662993683622.
## The mean MSE for 100 epochs is 77.90864362902043.
Observation: The MSE decreased significantly when increasing the number of epochs, indicating better performance and fitting of the model with more training iterations.
# Standard Deviation of MSE:

## For 50 epochs, the standard deviation is 11.836501926494584.
## For 100 epochs, the standard deviation is 14.80209886276834.
Observation: The slight increase in standard deviation suggests that the model's performance varies slightly more across the 50 runs at 100 epochs. This could be due to overfitting in some cases, depending on the complexity of the model and data.